nltk/app/srparser_app.py

```diff
@@ -59,9 +59,7 @@
       [h]\t Help
       [Ctrl-p]\t Print
       [q]\t Quit
-"""
 
-"""
```

Comment:
I'm not sure it was intended to have "Possible future improvements" as a module docstring; it looks more like a comment.